### ISSUE TITLE
[Python], random errors while trying to install pithon weels

### ISSUE THREAD

**Issue by falisca** [synthizer/synthizer#82]
Hi, I'm encountering strange and random errors installing the Python wheels, version 0.12.1
Seems that the Cython compilation can't find some header files, different ones on every run.
The problem occured in a new virtual environment, on Windows. I'm thinking it could be some configuration problem on my machine but can't tell what...
"synthizer-vendored\include\synthizer/audio_output.hpp(3): fatal error C1083: can't open include file: 'atomic': No such file or directory"
or on a previous run it gived:
"synthizer-vendored\include\synthizer/biquad.hpp(5): fatal error C1083: can't open include file: 'memory': No such file or directory"

I attach pip log for reference.

After the installation aborts pip installs the previous version, that installs fine but give me errors at instantiation time. That is matter for an other issue.
Please note that previously I succesfully used Synthizer in some projects, don't remember what version.

Any ideas?
Cheers.
[pip3.log](https://github.com/synthizer/synthizer/files/7854003/pip3.log)

**Comment by chrisnorman7** [comment]
Hi,
It might be worth ensuring build tools are in your path.

If you've got Visual Studio installed, you can get that by running the
vcvars64.bat script, which will be located somewhere like `C:\Program
Files\Microsoft Visual
Studio\2022\Community\VC\Auxiliary\Build\vcvars64.bat`.

This is pure guesswork though, I was under the impression that Python
wheels included all binary components so you don't have to build them
yourself.

Take care,

Chris Norman

**Comment by ahicks92** [comment]
Yeah. The Python wheels shouldn't require any building.  Are you sure it's deciding to use a wheel?  I suspect maybe you're on an older Python than I now publish wheels for, and aren't set up to build C extensions.  Upgrading your Python may be a place to start if it's old.

Also, moving this to the Python repo; Python issues should be opened there.
